interfaces/cython/cantera/test/test_convert.py

```diff
@@ -1064,7 +904,7 @@ def test_diamond(self):
 
     def test_lithium_ion_battery(self):
         name = 'lithium_ion_battery'
-        self.convert(name, self.cantera_data_path)
```

Comment:
Why are these paths being removed?

Reply:
Because the CTI files that are being converted are no longer in the main data folder -- any CTI / XML files that we still need for testing the converters have been moved to `test/data`, and since that's the only location that matters, it's now built into the `convert` function.